portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase

```diff
@@ -20,6 +20,43 @@ definition {
 		}
 	}
 
+	@priority = "5"
+	test CheckAUIPreloadConfiguration {
+		property test.name.skip.portal.instance = "UIInfrastructureUsecase#CheckAUIPreloadConfiguration";
```

Comment:
Let's not skip portal instance as much as possible. We'll need to include a proper teardown for this testcase. I realize the other testcases don't have a proper teardown as well, will need to address in a separate ticket.